packages/ui/src/components/DateInput/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -4572,7 +4572,7 @@ exports[`DateInput renders correctly with date-fns locale ru 1`] = `
                   <p
                     class="e17s8tvr0 cache-1re7cvs-StyledText-StyledText e1tg3t120"
                   >
-                    январь 2021 г.
+                    январь 2021 г.
```

Comment:
Bad node version ?

Reply:
Yep, je suis en 18.14.2 mais normalement @matthprost a fait une MR pour fix ca 🤔

Reply:
@BABAK0T0 you should use 18.15 instead